garrysmod/lua/vgui/dhtml.lua

```diff
@@ -63,6 +63,73 @@ function PANEL:Call( js )
 	self:QueueJavascript( js )
 end
 
+local function BuildFunction( func, ... )
+
+	-- Build a Javascript-safe JS function call.
+	-- To be run by either panel:RunFunction or panel:QueueFunction
+	--
+	-- First parameter is the JS function name,
+	-- additional parameters are the values to provide to the function call.
+	-- The values provided to the function call are treated as strings except where noted below.
+	
+	local args, amount = table.Pack( ... )

```

Comment:
wouldn't it not just be `local args = {...}` instead, as `amount` seems to be used.